Fundamentals/first_letter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+
```

Comment:
In our Python code style we use single new line between function definitions, in contrary to PEP8 style which uses 2 new lines. You have several places in the code with 2 newlines instead of 1, please fix that.